src/supermarket/app/controllers/api/v1/cookbook_versions_controller.rb

```diff
@@ -112,4 +117,13 @@ def require_collaborator_params
     params.require(:collaborator_failure)
     params.require(:collaborator_feedback)
   end
+
+  def create_metric(cookbook_version, quality_metric, failure, feedback)
+    MetricResult.create!(
+      cookbook_version: cookbook_version,
+      quality_metric: quality_metric,
+      failure: failure,
+      feedback: feedback
+    )
+  end
```

Comment:
What are the advantages to wrapping the call to `MetricResult.create!` in a controller method over calling it directly in the controller action?